.github/workflows/test.yaml

```diff
@@ -1183,7 +1183,7 @@ jobs:
 
   driver-adapters:
     needs: [detect_jobs_to_run]
-    if: contains(fromJSON(needs.detect_jobs_to_run.outputs.jobs), 'driver-adapters')
+    if: false && contains(fromJSON(needs.detect_jobs_to_run.outputs.jobs), 'driver-adapters')
```

Comment:
We should figure out a way how one can add a new project to the test.yaml without needing to do this here.

Reply:
Tech debt issue?

Reply:
https://github.com/prisma/team-orm/issues/676